File: Proyectos_android_studio-main/MyApplicationClase3/app/src/main/java/com/example/myapplicationclase3/MainActivity.java
package com.example.myapplicationclase3;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        final EditText txtNombre=(EditText) findViewById(R.id.TxtName);
        final EditText txtCarnet=(EditText) findViewById(R.id.TxtCarnet);
        final EditText txtCum=(EditText) findViewById(R.id.TxtCum);
        final Button btnCum=(Button) findViewById(R.id.BtnCum);

        btnCum.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
               Intent intent= new Intent(MainActivity.this,FrmCum.class);
                Bundle b = new Bundle();
                b.putString("NOMBRE",txtNombre.getText().toString());
                b.putString("CARNET",txtCarnet.getText().toString());
                b.putString("CUM",txtCum.getText().toString());
                intent.putExtras(b);
                startActivity(intent);
            }
        });
    }
}

File: Desafio1__DSM-eduardo/app/src/main/java/com/example/desafio1_dsm/MainActivity.kt
package com.example.desafio1_dsm

import android.content.DialogInterface
import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.app.AlertDialog
import android.widget.Button
import android.widget.EditText

lateinit var txtUser : EditText
lateinit var txtPass : EditText
lateinit var btnAcceder : Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        //Importando componentes de la Actividad
        txtUser = findViewById(R.id.txtUser)
        txtPass = findViewById(R.id.txtPass)
        btnAcceder = findViewById(R.id.btnAcceder)

        //Creando mensaje de error
        val builder : AlertDialog.Builder = AlertDialog.Builder(this)
        builder.setTitle("Error en el ingreso a la aplicacion")
        builder.setMessage("El nombre del usuario y/o contra estan erroneos")
        builder.setIcon(R.drawable.ic_launcher_background)
        builder.setPositiveButton("OK",DialogInterface.OnClickListener { dialog, i -> dialog.dismiss() })

        val alertDialog:AlertDialog = builder.create()

        //Boton acceder a la aplicacion
        btnAcceder.setOnClickListener{
            var user : String = txtUser.text.toString()
            var pass : String = txtPass.text.toString()
            if (user == "admin" && pass == "123456"){
                val intent = Intent(this, SelectActivity::class.java)
                startActivity(intent)
            } else {
                alertDialog.show()
            }
        }
    }
}